framework-plugins/lisk-framework-monitor-plugin/src/types.ts

```diff
@@ -50,16 +50,8 @@ export interface SharedState {
 		forkEventCount: number;
 		blockHeaders: Record<string, BlockHeader>;
 	};
-	transactions: {
-		transactions: Record<string, TransactionPropagationStats>;
-		averageReceivedTransactions: number;
-		connectedPeers: number;
-	};
-	blocks: {
-		blocks: { [key: string]: BlockPropagationStats };
-		averageReceivedBlocks: number;
-		connectedPeers: number;
-	};
+	transactions: Record<string, TransactionPropagationStats>;
```

Comment:
Why you changed the type here if the method seems to be returning the same properties that the original type was returning?

Reply:
type of `transactions` object is still unchanged which was also `Record<string, TransactionPropagationStats>` before, I only removed `averageReceivedTransactions` and `connectedPeers` because their value in `this._state.transactions` was never updated